team.md

```diff
@@ -5,39 +5,37 @@
 - Experience:
 - Interests:
 - Skills:
-- Twitter: [@__kfv__](https://twitter.com/__kfv__)
 - Timezone: ?
+- Twitter: [@__kfv__](https://twitter.com/__kfv__)
+
 
 ## Foo, Ji-Haw
 - About: Tinkers in solar, ESP32. Used to deploy FreeBSD commercially; now runs them personally in the garage.
 - Experience:
 - Interests:
 - Skills:
-- Twitter: [@KungFoo10](https://twitter.com/kungfoo10)
 - Timezone: Central Time, US (GMT -0600)
+- Twitter: [@KungFoo10](https://twitter.com/kungfoo10)
 
 ## Ken (khbsd_)
-  - About: Massive computer nerd. Loves old computers, UNIX, and BSD. Is 10% goofier than you.
-  - Experience: Telecom Field Tech, hobbyist computer enthusiast/programmer.
-  - Interests: 80s computers, UNIX, Programming, Infosec.
-  - Skills: Proficient in Python, Java, HTML and CSS. Can probably pick up whatever else is needed. Desktop Expert, customizer.
-  - Twitter: [@khbsd_](https://twitter.com/khbsd_)
-  - Timezone: CST, (GMT -0600)
+- About: Massive computer nerd. Loves old computers, UNIX, and BSD. Is 10% goofier than you.
+- Experience: Telecom Field Tech, hobbyist computer enthusiast/programmer.
+- Interests: 80s computers, UNIX, Programming, Infosec.
+- Skills: Proficient in Python, Java, HTML and CSS. Can probably pick up whatever else is needed. Desktop Expert, customizer.
+- Timezone: CST, (GMT -0600)
```

Comment:
- Use consistent TZ format: `UTC[+-]XXXX` for all entries. Eg: My entry below is `UTC+XX`
- Probably worth putting country in, as timezone names alone remain a bit ambiguous (USA and AU has "EST" for example)

Reply:
good catch, just a sec

Reply:
Done!